Sources/ShortcutFoundation/Extensions/Date+Extension.swift

```diff
@@ -22,6 +22,13 @@ public extension Date {
     static var now: Date {
         return Date()
     }
+    
+    static func convertToDate(inputDate: String) -> Date {
```

Comment:
This function has to supply witch format or specify in the name of the function that it transforms with this specific format. In all other cases than having exactly "2020-11-22" format, this function will fail

Reply:
Fixed in 252c7aa